src/lib/index.js

```diff
@@ -87,6 +92,13 @@ lib.pushUnique = require('./push_unique');
 
 lib.cleanNumber = require('./clean_number');
 
+lib.num = function num(v) {
```

Comment:
Do we want a more descriptive name for this now that it's not just internal to one module? `ensureNumber`? `safeNumber`?